src/sentry/lang/native/plugin.py

```diff
@@ -209,10 +209,14 @@ def process_frame(self, processable_frame, processing_task):
                 symbolicated_frames = self.sym.symbolize_frame(
                     instruction_addr,
                     self.sdk_info,
-                    symbolserver_match=processable_frame.data['symbolserver_match']
+                    symbolserver_match=processable_frame.data['symbolserver_match'],
+                    trust=raw_frame.get('trust'),
                 )
                 if not symbolicated_frames:
-                    return None, [raw_frame], []
+                    if raw_frame.get('trust') == 'scan':
```

Comment:
Should `scan` have a 'constant'?

Reply:
It's okay as such. We generally don't have constants for such values at the moment.